Public/Src/Engine/UnitTests/Processes.Detours/FileAccessManifestTreeTest.cs

```diff
@@ -606,7 +606,10 @@ public void TestSolution10000()
 
         private static void TestManifestRetrieval(IEnumerable<ValidationData> validationData, FileAccessManifest fam)
         {
-            string famDescription = Environment.NewLine + string.Join(Environment.NewLine, fam.Describe());
+            foreach (var line in fam.Describe())
+            {
+                Console.WriteLine(line);
+            }

```

Comment:
It's fine to print this unconditionally up-front.

Reply:
Or even not print it at all

Reply:
We need it for debugging in case of error.